examples/oom_fuzz_test.cc

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ *Licensed under the Apache License, Version 2.0 (the "License");
+ *you may not use this file except in compliance with the License.
+ *You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *Unless required by applicable law or agreed to in writing, software
+ *distributed under the License is distributed on an "AS IS" BASIS,
+ *WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *See the License for the specific language governing permissions and
+ *limitations under the License.
+ */
+
+// A fuzz target that creates a memory leak and causes OOM errors.
+
+#include <stdint.h>
+#include <stddef.h>
+
+void leakMemory() {
+    int* zombiePtr = new int(100);
+    return;
+}
+
+void doOOM(const uint8_t *Data, size_t DataSize) {
```

Comment:
Use `DoOom` here. Acronyms should have first letter capitalized, but not the rest, as per https://google.github.io/styleguide/cppguide.html#Variable_Names